src/pykeen/models/unimodal/mure.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of MuRE."""
+
+from typing import Any, ClassVar, Mapping
+
+from ..nbase import ERModel
+from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
+from ...nn import EmbeddingSpecification
+from ...nn.init import xavier_normal_
+from ...nn.modules import MuREInteraction
+from ...typing import Hint, Initializer
+
+__all__ = [
+    'MuRE',
+]
+
+
+class MuRE(ERModel):
+    r"""An implementation of MuRE from [balazevic2019b]_.
+
+    ---
+    citation:
+        author: Balažević
+        year: 2019
+        link: https://arxiv.org/abs/1905.09791
+    """
+
+    #: The default strategy for optimizing the model's hyper-parameters
+    hpo_default: ClassVar[Mapping[str, Any]] = dict(
+        embedding_dim=DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE,
+        p=dict(type=int, low=1, high=2),
+    )
+
+    def __init__(
+        self,
+        embedding_dim: int = 200,
```

Comment:
@mberr need to check in on these

Reply:
@cthoyt I updated them. Now, the initializers cannot be provided by the constructor. A novelty here is that we have more than one initializer for entities/relations since there are more than one representation for them.